index.js

```diff
@@ -24,12 +24,31 @@ try {
   const logoPosition = core.getInput('logoPosition');
   const style        = core.getInput('style');
   const cacheSeconds = core.getInput('cacheSeconds');
+  const valColorRange = core.getInput('valColorRange');
+  const minColorRange = core.getInput('minColorRange');
+  const maxColorRange = core.getInput('maxColorRange');
+  const invertColorRange = core.getInput('invertColorRange');
 
   if (labelColor != '') {
     content.labelColor = labelColor;
   }
 
-  if (color != '') {
+  if (minColorRange != '' && maxColorRange != '' && valColorRange != '') {
+    var max = parseFloat(maxColorRange);
+    var min = parseFloat(minColorRange);
+    var val = parseFloat(valColorRange);
+    if (val < min) val = min;
+    if (val > max) val = max;
+    var hue = 0
+    if (invertColorRange == '') {
+      hue = Math.floor((val - min) / (max - min) * 100);
+      content.message = val + " / " + max + " " + content.message
+    } else {
+      hue = Math.floor((max - val) / (max - min) * 100);
+      content.message = valColorRange + " " + content.message
+    }
+    content.color = "hsl(" + hue + ", 100%, 50%)";
```

Comment:
Talking about colors, right after opening the PR I started thinking the 50% of lightness is too bright. What do you think about decreasing it to 40%?
```suggestion
    content.color = "hsl(" + hue + ", 100%, 40%)";
```

Reply:
Maybe we could make it configurable? Like with a `colorRangeSaturation` and a `colorRangeLightness` parameter?

Reply:
Good idea. But, for this PR, can we close with just the current modifications? Adding more parameters using this interface will be a little cumbersome. I will add them using another fork afterward (or any other way you prefer)